openmeter/ent/schema/lock.go

```diff
@@ -0,0 +1,38 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/dialect"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type LockTable struct {
```

Comment:
Please remove the table definition too. Otherwise looks good.

Reply:
Thanks for catching, updated now